test/unit/gpu/test-traversal.cxx

```diff
@@ -0,0 +1,283 @@
+/*
+ * Copyright (c) 2016, Lawrence Livermore National Security, LLC.
+ *
+ * Produced at the Lawrence Livermore National Laboratory.
+ *
+ * All rights reserved.
+ *
+ * For release details and restrictions, please see RAJA/LICENSE.
+ */
+
+#include <cfloat>
```

Comment:
This file doesn't currently compile (see comment above on test/unit/gpu/CMakeLists.txt)